### PR TITLE
BUX-543: Set Tall as Default

### PR DESCRIPTION
**Pull request by Nazarii-4chain**
Updated configs and Readme
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/BuxOrg/bux-server/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/BuxOrg/bux-server/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: config/defaults.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func getDefaultAppConfig() *AppConfig {
 
 func getAuthConfigDefaults() *AuthenticationConfig {
 	return &AuthenticationConfig{
-		AdminKey:        "xpub661MyMwAqRbcFgfmdkPgE2m5UjHXu9dj124DbaGLSjaqVESTWfCD4VuNmEbVPkbYLCkykwVZvmA8Pbf8884TQr1FgdG2nPoHR8aB36YdDQh",
+		AdminKey:        "xpub661MyMwAqRbcEp7YgDpGXquSF2NW3GBAU3SXTikFT1nkxHGbxjG9RgGxr9X3D4AYsJ6ZqYjMGcdUsPDQZoeibKECs5d56f1w9rfF3QrAAu9",
```
Comment (wregulski) [inline config/defaults.go:31]: Why did you change the default AdminKey? It is related to bux-wallet-backend one.

File: config.example.yaml [hunk 2]
```diff
@@ -86,8 +86,11 @@ nodes:
   protocol: arc
   # list of apis used for getting and broadcasting transactions
   apis:
-    - arc_url: https://arc.gorillapool.io
-      token:
+    # gorillapool can be used as well
+#    - arc_url: https://arc.gorillapool.io
+#    - token: ""
+    - arc_url: https://tapi.taal.com/arc/v1
```
Comment (wregulski) [inline config.example.yaml:92]: We should use `https://tapi.taal.com/arc/v1`
without the "t" letter.

**Review by wregulski — CHANGES_REQUESTED** [review]

File: config.example.yaml (hunk 2 above)
Comment (Nazarii-4chain) [reply config.example.yaml:92]: Found endpoint here 
https://docs.taal.com/core-products/transaction-processing/arc-endpoints-beta

Comment (Nazarii-4chain) [reply config.example.yaml]: Changed

File: config/defaults.go (hunk 1 above)
Comment (Nazarii-4chain) [reply config/defaults.go]: Reverted

**Review by wregulski — APPROVED** [review]

**Review by chris-4chain — APPROVED** [review]
